tasks/set_vars.yml

```diff
@@ -1,21 +1,27 @@
 ---
-- name: Ensure ansible_facts used by role
-  setup:
-    gather_subset: min
-  when: not ansible_facts.keys() | list |
-    intersect(__template_required_facts) == __template_required_facts
+- name: "Gather the packages facts"
+  ansible.builtin.package_facts:
+    manager: rpm
```

Comment:
omit `manager` here

Reply:
please leave the `setup` and `Set platform/version specific variables` in this file